README.md

```diff
@@ -168,6 +168,10 @@ flank:
   ## Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
   # local-result-dir: flank
 
+  ## Keeps the full path of downloaded files. Required when file names are not unique.
+  ## Default: false  
+  # keep-file-path: false
```

Comment:
Make sure to update `flank.ios.yml`
https://github.com/Flank/flank/blob/master/test_runner/flank.ios.yml

Reply:
Thanks for pointing this out!